MANUAL.txt

```diff
@@ -3027,6 +3027,36 @@ This extension can be enabled/disabled for the following formats:
 output formats
 :  `odt`, `opendocument`
 
+#### Extension: `references_over_links` ####
+
+Links to headings, figures and tables inside the document are
+substituted with (cross-)references. References use the text
+of the referenced document element such as the heading text,
+figure or table caption. The original link text is replaced once
+the generated document is refreshed.
+
+This extension can be enabled/disabled for the following formats:
+
+output formats
+:  `odt`, `opendocument`
+
+#### Extension: `number_prefix_references` ####
+
+Add a number prefix to references within the document that lack one.
+For example a first heading of "Introduction" that is number prefixed
+in the final document as "1 Introduction" will in references also be
+number prefixed as "1 Introduction".
+
+This typically requires the final document to have indexes refreshed
+in a native editor such as libreoffice as pandoc is not aware of the
+number style of the document.
+
+This further assumes that references are used within the document
+instead of links - see the `references_over_links` extension.
+
+output formats
+:  `odt`, `opendocument`
+
```

Comment:
I'm concerned that a reader might think "references" refers to markdown link references or something like that.  Lots of formats have things called "references."

Would using the term "crossreferences" or something like "xrefs" be clearer?  Maybe the extension should be `crossrefs` and `numbered_crossrefs`?  (The latter could automatically enable use of cross-references, so people wouldn't have to specify both.)

Do you think these would apply to other formats besides opendocument/docx?

It may be that these names are the best, but I think we should hash out alternatives first, because this is an API change and hard to reverse.

Reply:
What about `xrefs_for_internal_links` and `numbered_xrefs` ? I think that should be very clear - but my perception is slanted.

This should be applicable to OpenDocument, Docx most probably, and DocBook output formats. At least.

Reply:
Yes, I like that better.

Reply:
One thing that occurs to me, though, is that with `xrefs_for_internal_links` one might expect that *all* internal links would become xrefs.  `xrefs` by itself doesn't imply this and may for that reason be better (also shorter).  I think you'd probably need to look in the manual to figure out what it does, but that is probably true regardless of the name.

Reply:
Ok - `xrefs` and `numbered_xrefs` it is. And `numbered_xrefs` enables `xrefs`. Thanks for your patience.